lib/rules/named-grid-areas-no-invalid/index.js

```diff
@@ -60,14 +58,14 @@ function rule(actual) {
 			if (reportSent) return;
 
 			if (!isRectangular(areas)) {
-				complain(messages.notRectangular());
+				complain(messages.expectedSameNumber());
```

Comment:
Should I add a `return`? is it worth checking named areas are rectangular and contiguous if the the grid itself is not rectangular?

Reply:
I've decided to make this change to `return` if strings do not not have same number of tokens (grid not rectangular). Let me know if you think this is bad in anyway.